package proto_ff;

import "yd_fieldoptions.proto";


message chooseGiftchooseGiftrechargeDesc
{
	optional int64 Box = 1[(yd_fieldoptions.field_cname) = "奖励"];
	optional int32 Id = 2[(yd_fieldoptions.field_cname) = "计费点"];
}

message chooseGiftchooseGift
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int64 box = 2[(yd_fieldoptions.field_cname) = "礼包奖励"];
	optional int32 switchTime = 3[(yd_fieldoptions.field_cname) = "切换时间"];
	repeated chooseGiftchooseGiftrechargeDesc recharge = 4[(yd_fieldoptions.field_cname) = "充值", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_chooseGiftchooseGift
{
	repeated chooseGiftchooseGift chooseGiftchooseGift_List = 1[(yd_fieldoptions.field_arysize)=105];
}
